packages/expect/src/spy_matchers.js

```diff
@@ -230,4 +268,20 @@ const formatMismatchedArgs = (expected, received) => {
   return printedArgs.join('\n');
 };
 
+const isPositiveInteger = (integer: number) => {
```

Comment:
Can you inline it? I'm also not sure why you have to call `parseInt` here? the `nth < 0` check above seems to work just fine?

Reply:
Nevermind, you want an integer, not a number. Still, can you inline it?